README.md

```diff
@@ -7,109 +7,4 @@ The community has created this knowledge base to help you **learn** and **grow**
 
 - Visit the [Glossary](GLOSSARY.md) to understand more about a specific term.
 
-## Resources
-
-- [Solana: How to send custom instructions via instruction data](https://dev.to/cogoo/solana-how-to-send-custom-instructions-via-instruction-data-4g9g)
-
-  Author: Colin Ogoo
-  Walk through the process of sending a custom instruction to a solana on-chain program. We'll modify the solana example helloworld to take two instructions, SayHello and SayGoodbye.
-
-- [How to write your first decentralized app - scaffold-eth Challenge 1: Staking dApp](https://dev.to/stermi/scaffold-eth-challenge-1-staking-dapp-4ofb)
```

Comment:
I removed this just to test, but I'm sure when you run it, it'll produce the content.